server/lib/genome/groupers/drug_grouper.rb

```diff
@@ -34,63 +38,173 @@ def run(source_id = nil)
             normalized_id = normalized_drug
           else
             normalized_id = normalized_drug['therapy_descriptor']['therapy_id']
-            create_new_drug normalized_drug['therapy_descriptor'] if Drug.find_by(concept_id: normalized_id).nil?
+            create_new_drug(normalized_drug['therapy_descriptor']) if Drug.find_by(concept_id: normalized_id).nil?
           end
           add_claim_to_drug(drug_claim, normalized_id)
         end
       end
 
+      def create_sources
+        drug_source_type = SourceType.find_by(type: 'drug')
+
+        source_meta = fetch_source_meta
+
+        rxnorm = Source.where(
+          source_db_name: 'RxNorm',
+          source_db_version: source_meta['RxNorm']['version'],
+          base_url: 'https://rxnav.nlm.nih.gov/REST/rxcui/rxcui/allrelated.xml',
+          site_url: 'https://www.nlm.nih.gov/research/umls/rxnorm/overview.html',
```

Comment:
this function looks good. I agree it doesn't make sense to include here the sources we already have importers for